src/app/gallery/gallery.component.ts

```diff
@@ -1,18 +1,28 @@
-import { Component, OnInit } from '@angular/core';
-import { featuredHouses, featuredRooms } from 'src/assets/data/featured';
-import { FeaturedSimsInt } from 'src/assets/interfaces/FeaturedSimsInt';
+import { Component, OnInit } from "@angular/core";
+import { featuredHouses, featuredRooms } from "src/assets/data/featured";
+import { FeaturedSimsInt } from "src/assets/interfaces/FeaturedSimsInt";
```

Comment:
```suggestion
import { Component, OnInit } from "@angular/core";

import { featuredHouses, featuredRooms } from "../../assets/data/featured";
import { FeaturedSimsInt } from "../../assets/interfaces/FeaturedSimsInt";
```

Resolves the unresolved path error.